utility/src/main/res/values/dimens.xml

```diff
@@ -3,7 +3,9 @@
   <dimen name="bullet_radius">4dp</dimen>
   <dimen name="bullet_gap_width">16dp</dimen>
   <dimen name="bullet_y_offset">2dp</dimen>
-  <dimen name="bullet_leading_margin">24dp</dimen>
+  <dimen name="bullet_x_offset">24dp</dimen>
+  <dimen name="bullet_leading_margin">20dp</dimen>
```

Comment:
Please add a comment here & below to explain what these margins are.

Also, perhaps better names might be: ``spacing_between_bullet_and_text`` and ``spacing_before_bullet`` to be really clear.

Reply:
Done